aspnetcore/fundamentals/servers/kestrel/http3.md

```diff
@@ -62,13 +62,7 @@ HTTP/3 is discovered as an upgrade from HTTP/1.1 or HTTP/2 via the `alt-svc` hea
 
 ## Get started
```

Comment:
Update the Windows requirement to include Windows Server 2022.

Reply:
Done